macros/GroupData.json

```diff
@@ -1258,6 +1258,93 @@
             "events":     [ "shippingaddresschange",
                             "shippingoptionchange" ]
         },
+        "Performance API": {
+            "overview":   [ "Performance API" ],
+            "guides":     [ { "url": "/en-US/docs/Web/API/Frame_timing_API",
+                              "title": "Frame Timing API" },
+                            { "url": "/en-US/docs/Web/API/Long_Tasks_API",
+                              "title": "Long Tasks API" },
+                            { "url": "/en-US/docs/Web/API/Navigation_timing_API",
+                              "title": "Navigation Timing API" },
+                            { "url": "/en-US/docs/Web/API/Navigation_timing_API/Using_Navigation_Timing",
```

Comment:
https://developer.mozilla.org//en-US/docs/Web/API/Navigation_timing_API/Using_Navigation_Timing 
This page page looks useless. Are you planning to write something here? If not, I think it shouldn't be here for now.